python/ray/train/constants.py

```diff
@@ -82,6 +82,10 @@ def _get_defaults_results_dir() -> str:
 # or Train worker to the trial directory. Defaults to 1.
 RAY_CHDIR_TO_TRIAL_DIR = "RAY_CHDIR_TO_TRIAL_DIR"
 
+# Set this to 1 to count preemption errors toward `FailureConfig(max_failures)`.
+# Defaults to 0, which always retries on node preemption failures.
+RAY_TRAIN_COUNT_PREEMPTION_ERRORS = "RAY_TRAIN_COUNT_PREEMPTION_ERRORS"
```

Comment:
nit: Maybe rename this as `RAY_TRAIN_COUNT_PREEMPTION_AS_FAILURE` to make it more explicit